grunnbelop/src/test/kotlin/no/nav/dagpenger/grunnbelop/GrunnbelopTest.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.dagpenger.grunnbelop
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.Month
+import java.time.YearMonth
+
+class GrunnbelopTest {
+
+    @Test
+    fun ` Skal returnere grunnbeløp på 99858 for måned april 2020 `() {
+
+        Assertions.assertEquals(99858.toBigDecimal(), getGrunnbeløpForMåned(YearMonth.of(2019, Month.MAY)).verdi)
```

Comment:
skal det være   `YearMonth.of(2020, Month.MAY)` ikke sant?

Reply:
nei, men det skal være of(2020, Month.APRIL). Den gjelder fra og med mai 19 til og med april 20. Så tester mai 19 under, og planen var å teste april 20 her. Skal fikse